.eslintrc

```diff
@@ -0,0 +1,4 @@
+extends: "xo-space/esnext"
```

Comment:
It's an `eslint` config:

https://github.com/sindresorhus/eslint-config-xo-space